composeApp/src/commonMain/kotlin/com/tajemniktv/tajsos/ui/screens/tasks/detail/TaskDetailBlocks.kt

```
@@ -940,7 +940,13 @@ private fun RenderTaskMetadataProperties(
                 TaskPropertyOption(TaskDuePreset.Tomorrow.name, "Tomorrow"),
                 TaskPropertyOption(TaskDuePreset.InSevenDays.name, "In 7 days"),
             ),
-        onSelect = { value -> context.onDuePresetChange(TaskDuePreset.valueOf(value)) },
+        onSelect = { value ->
+            try {
+                context.onDuePresetChange(TaskDuePreset.valueOf(value))
+            } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)